src/features/war/components/WarCollection.tsx

```diff
@@ -21,60 +25,133 @@ export const WarCollection: React.FC<{ side: WarSide }> = ({ side }) => {
+                style={{
+                  width: `${PIXEL_SCALE * 86}px`,
+                }}
+                onClick={() => setShowModal(true)}
+              />
+              <img
+                src={femaleGoblin}
+                className="absolute left-20 -bottom-2 cursor-pointer hover:img-highlight"
+                style={{
+                  width: `${PIXEL_SCALE * 16}px`,
+                }}
+                onClick={() => setShowModal(true)}
+              />
+              <img
+                src={maleGoblin}
+                className="absolute right-12 -bottom-2 cursor-pointer hover:img-highlight"
+                style={{
+                  width: `${PIXEL_SCALE * 16}px`,
+                  transform: "scaleX(-1)",
+                }}
+                onClick={() => setShowModal(true)}
+              />
+            </div>
+            <div
+              className="absolute z-10"
               style={{
-                width: `${PIXEL_SCALE * 16}px`,
-                transform: "scaleX(-1)",
+                left: `${GRID_WIDTH_PX * 61}px`,
+                top: `${GRID_WIDTH_PX * 30.5}px`,
               }}
-              onClick={() => setShowModal(true)}
-            />
+            >
+              <img
+                src={goblinFlags}
+                style={{
+                  width: `${PIXEL_SCALE * 60}px`,
+                }}
+              />
+            </div>
+            <div
+              className="absolute z-10"
+              style={{
+                left: `${GRID_WIDTH_PX * 56.9}px`,
+                top: `${GRID_WIDTH_PX * 34.6}px`,
+              }}
+            >
+              <img
+                src={weapons}
+                style={{
+                  width: `${PIXEL_SCALE * 29}px`,
+                }}
+              />
+            </div>
           </>
         )}
         {side === WarSide.Human && (
           <>
-            <img
-              src={femaleHuman}
-              className="absolute left-20 -bottom-2 cursor-pointer hover:img-highlight"
+            <div
+              className="absolute z-10"
+              style={{
+                left: `${GRID_WIDTH_PX * 51.57}px`,
+                top: `${GRID_WIDTH_PX * 32.8}px`,
+              }}
+            >
+              <img
+                src={humanBase}
+                style={{
+                  width: `${PIXEL_SCALE * 86}px`,
+                }}
+              />
+              <img
+                src={femaleHuman}
+                className="absolute left-20 -bottom-2 cursor-pointer hover:img-highlight"
+                style={{
+                  width: `${PIXEL_SCALE * 16}px`,
+                }}
+                onClick={() => setShowModal(true)}
+              />
+              <img
+                src={maleHuman}
+                className="absolute right-12 -bottom-2 cursor-pointer hover:img-highlight"
+                style={{
+                  width: `${PIXEL_SCALE * 16}px`,
```

Comment:
The humans now look a bit too big. The `PIXEL_SCALE` doesn't work the same as land expansion unfortunately. We will need to hardcode them a bit smaller to match the other NPCs

<img width="397" alt="Screen Shot 2022-08-16 at 3 36 05 pm" src="https://user-images.githubusercontent.com/11745561/184805393-d4ae0f15-d123-4bff-aeaa-e9c2ef6e80a9.png">